app/src/main/java/projekt/substratum/common/commands/FileOperations.java

```diff
@@ -408,8 +409,8 @@ public static boolean copyFileOrDir(AssetManager assetManager, String listDir,
 
     private static boolean copyFile(AssetManager assetManager, String filename,
                                     String destination, String remember) {
-        InputStream inputStream;
-        OutputStream outputStream;
+        InputStream inputStream = null;
```

Comment:
it didnt give any warning about redundancy of these checks but
i can remove them if you want although